fapolicy_analyzer/ui/fapd_manager.py

```diff
@@ -54,41 +58,43 @@ def __init__(self, bSuFapdControl=True):
         self.initial_daemon_status()
 
         # ToDo: Verify that FAPD_LOGPATH works in the pkexec environment
-        if os.environ.get("FAPD_LOGPATH"):
-            self.fapd_profiling_stdout = os.environ.get("FAPD_LOGPATH") + ".stdout"
-            self.fapd_profiling_stderr = os.environ.get("FAPD_LOGPATH") + ".stderr"
+        fapd_logpath_tmp = None
+        fapd_logpath_tmp = os.environ.get("FAPD_LOGPATH")
+        if fapd_logpath_tmp and len(fapd_logpath_tmp):
```

Comment:
See above wrt that env var being always set. I'll verify it works in both the dev env and when using the wrapper script w/pkexec.

Reply:
```suggestion
        fapd_logpath_tmp = os.getenv("FAPD_LOGPATH")
        if fapd_logpath_tmp:
```
That should work.  `os.getenv` will return the default of None if the env is not available.

Reply:
Yeah, agreed but in the pkexec runtime env that env var will always be set even if it's a Null string so None is never returned from getenv (it returns ""). It will work correctly in the dev env though. But your statement about a string with zero length = False allows things to work correctly in either dev or deployed (pkexec) env, so I used that approach. 

> since empty string is also a falsy value.

Thx for that. Hopefully it's good now.

Reply:
Verified in both dev and rpm deployed environments.